tests/test_tutorial/test_cookie_param_models/test_tutorial002.py

```diff
@@ -71,9 +77,8 @@ def test_cookie_param_model_invalid(client: TestClient):
                     }
                 ]
             }
-        )
-        | IsDict(
-            # TODO: remove when deprecating Pydantic v1
```

Comment:
We should keep this TODO statement somehow

Reply:
Can we add it to the function? 

"Remove v1 argument when deprecating Pydantic v1"

Maybe we can use [codecrumbs](https://15r10nk.github.io/codecrumbs/introduction/) in the future to clean things up. 

There are a lot of refactorings missing which would be needed for this. But it would be a reason to continue this project :-)